### ISSUE TITLE
Types defs from npm cache conflict with Deno types

### ISSUE THREAD

**Issue by jollytoad** [denoland/deno#24671]
Version: Deno 1.45.2

I've starting seeing errors from TS checking that says the `ReadableStream.from` property does not exist.

This is a complete error:

```
TS2339 [ERROR]: Property 'from' does not exist on type '{ new (underlyingSource: UnderlyingByteSource, strategy?: { highWaterMark?: number | undefined; } | undefined): ReadableStream<Uint8Array>; new <R = any>(underlyingSource: UnderlyingDefaultSource<...>, strategy?: QueuingStrategy<...> | undefined): ReadableStream<...>; new <R = any>(underlyingSource?: UnderlyingSourc...'.
  const body = ReadableStream.from(streamEvents())
                              ~~~~
    at file:///Users/mgibson/deno/jollytoad/app/routes/sse/feed.tsx:7:31
```

Code for this module is here: https://github.com/jollytoad/home/blob/main/app/routes/sse/feed.tsx#L7

Although I'm seeing many of these, here is the full log from my CI action: https://github.com/jollytoad/home/actions/runs/10041643107/job/27750190854#step:6:827

I also see the errors in VSCode too:

![Screenshot 2024-07-22 at 2 28 11 PM](https://github.com/user-attachments/assets/13346d59-4d3b-48f9-a064-f1ed7ca081ff)

It seems that the types from the npm cache are being leaked implicitly:

![Screenshot 2024-07-22 at 2 29 36 PM](https://github.com/user-attachments/assets/abc9b963-bd47-42a2-8e3a-008022f7d4f6)

**Comment by dsherret** [comment]
Looks like a duplicate of #24578

**Comment by jollytoad** [comment]
I can confirm this is now fixed in Deno 1.45.3
